search.py

```diff
@@ -50,11 +50,11 @@ def get_attrib_from_element(tree, tag, att_name):
 
 # Specific Functions
 
+
 def get_title(tree):
     """"return a list of each line of the title of the piece"""
```

Comment:
The docsting should describe the arguments and return values.

Reply:
nick will have the docstrings updated in his pull request